c++/src/kj/compat/http.c++

```diff
@@ -4739,118 +4753,123 @@ public:
         return httpOutput.flush().then([]() { return false; });
       }
 
-      KJ_IF_MAYBE(req, request) {
-        auto& headers = httpInput.getHeaders();
+      KJ_SWITCH_ONEOF(requestOrProtocolError) {
+        KJ_CASE_ONEOF(request, HttpHeaders::Request) {
```

Comment:
The diff gets screwed up here, but the only substantive change in this code block is the indentation change necessitated by using a `KJ_SWITCH_ONEOF` instead of a `KJ_IF_MAYBE`.

Reply:
FWIW, diff is easier to read if you click the gear icon in the bar at the top of the diff, then check "hide whitespace changes" and click "apply and reload".  I'm guessing there's a more permanent setting elsewhere.

Reply:
Oh, yeah, thanks! I remember setting that in the past and had thought it was persistent across PRs. Now I'm less confused.